spec.md

```diff
@@ -1915,10 +2050,11 @@ Bitwise operations:
 
 Concatenation
    string + string
+    bytes + bytes
      list + list
     tuple + tuple
 
-Repetition (string/list/tuple)
+Repetition (bytes/string/list/tuple)
```

Comment:
Nit: string first, then bytes, to match order for concatenation (and elsewhere)?

Reply:
Done.